lib/transformer.dart

```diff
@@ -327,8 +327,8 @@ class KbcCreditCardTransformer with Transformer {
     return YnabApiTransaction(
       accountId: account,
       amount: _createAmount(list[KbcCCColumn.amountInEUR.index]),
-      payeeName: list[KbcCCColumn.merchant.index],
-      memo: list[KbcCCColumn.explanation.index],
+      payeeName: list[KbcCCColumn.merchant.index].split('/').first,
```

Comment:
Does this fix the occasional "name" exception from YNAB?

Reply:
Hm, I'm not sure. Mostly strips the city name from for example `APPLE.COM BILL/Hollyhill`